### PR TITLE
Filter out country version blinking in Google Terms of Services (update)

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: services/Google.filters.js [hunk 1]
```diff
@@ -4,3 +4,23 @@ export function removeUTMfromUrls(document) {
     link.href = link.href.replace(/utm_[^=]*=[^&]*[&]{0,1}/g, '');
   });
 }
+
+export function removeCountryVersion(document) {
+  const countryVersionLink = document.querySelector('a[data-name="country-version"]');
+  const countryVersionTitle = document.querySelector('#footnote-country-version');
+  const disclaimerTitle = document.querySelector('#footnote-disclaimer');
+
+  console.log('countryVersionLink', countryVersionLink);
```
Comment (LucasVerneyDGE) [inline services/Google.filters.js:13]: Extra console.log?

File: services/Google.filters.js [hunk 2]
```diff
@@ -4,3 +4,23 @@ export function removeUTMfromUrls(document) {
...
+  if (countryVersionLink) {
+    // Remove "country version" link at top of the page
+    countryVersionLink.parentNode.remove();
+  }
+
+  if (countryVersionTitle && disclaimerTitle) {
+    // Remove all the "country version" section in the CGUs
+    const countryVersionSectionContent = document.createRange();
+    countryVersionSectionContent.setStartBefore(countryVersionTitle);
```
Comment (LucasVerneyDGE) [inline services/Google.filters.js:22]: Proposal:

```js
console.assert(countryVersionSectionContent.cloneContents().querySelectorAll('h3').length === 1, 'Country version selector seems to be too large.');
```

File: services/Google.filters.js (hunk 1 above)
Comment (Ndpnt) [reply services/Google.filters.js:13]: Oups 😬

File: services/Google.filters.js (hunk 2 above)
Comment (Ndpnt) [reply services/Google.filters.js:22]: What about directly throwing an error to avoid recording a incorrect version? :)

Comment (LucasVerneyDGE) [reply services/Google.filters.js:22]: :+1:

File: services/Google.filters.js [hunk 3]
```diff
@@ -4,3 +4,28 @@ export function removeUTMfromUrls(document) {
...
+  }
+
+  if (countryVersionTitle && disclaimerTitle) {
+    const countryVersionSectionContent = document.createRange();
+    countryVersionSectionContent.setStartBefore(countryVersionTitle);
+    countryVersionSectionContent.setEndBefore(disclaimerTitle);
+
+    // Avoid removing too much content
+    if (countryVersionSectionContent.cloneContents().querySelectorAll('h3').length > 1) {
+      throw new Error('Country version selection seems to be too large.');
```
[the remark this answers is not in the document]
Comment (Ndpnt) [reply services/Google.filters.js:25]: It's some sort of failsafe for unforeseen cases. If they add a section between the country version title and the disclaimer title.

**Pull request merged**